lib/server.js

```diff
@@ -287,6 +287,11 @@ function expiry(block2cache, k) {
 
 CoAPServer.prototype._handle = function(packet, rsinfo) {
 
+  if (packet.code[0] !== '0') {
```

Comment:
This section needs an explanation. Could you please add the following comment:
According to RFC7252 Section 4.2 receiving a confirmable messages that can't be processed, should be rejected by ignoring it AND sending a reset. In this case confirmable response message would be silently ignored, which is not exactly as stated in the standard. However, sending a reset would interfere with a coap client which is re-using a socket (see pull-request #131).